src/compiler/scanner.ts

```diff
@@ -429,6 +429,7 @@ namespace ts {
               case CharacterCodes.slash:
                   // starts of normal trivia
               case CharacterCodes.lessThan:
+              case CharacterCodes.bar:

```

Comment:
Please don't open a PR - we already consistently use `bar` and `Bar` throughout the codebase.